iquip/apps/builder.py

```diff
@@ -120,14 +126,18 @@ def run(self):
         
         Submits the experiment to the proxy server.
 
-        Regardless of whether it is successful or not, the server returns the run identifier.
-        After finished, the submitted signal is emitted.
+        If only the experiment was submitted well regardless of whether it runs successfully or not,
```

Comment:
I think "Whenever ..." is more appropriate. We use "If only ..." for the assumption expressions (of hopes).

Reply:
It looks better! I will apply it.